src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -3726,6 +3726,7 @@ private StructDeclarationSyntax DeclareStruct(TypeDefinitionHandle typeDefHandle
         {
             case "RECT":
             case "SIZE":
+            case "SYSTEMTIME":
```

Comment:
Given this now has special behavior, can you please add a test that generates only `SYSTEMTIME` and ensures no compiler diagnostics are produced? There are several similar tests already in existence for you to copy. In fact if we don't already have a `[Theory]` for all these templated APIs, could you add one theory method for all these templated APIs?